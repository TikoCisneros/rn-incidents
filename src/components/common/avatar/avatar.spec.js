import React from 'react';
import { shallow } from 'enzyme';
import Avatar from './avatar';

describe('Avatar component', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Avatar size={40} firstLetter="A" />);
    expect(wrapper).toMatchSnapshot();
  });
});