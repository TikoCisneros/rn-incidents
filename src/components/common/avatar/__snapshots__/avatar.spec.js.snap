// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar component renders correctly 1`] = `
<Styled(View)
  style={
    Object {
      "borderRadius": 20,
      "height": 40,
      "width": 40,
    }
  }
>
  <Styled(Styled(Text))
    type="HEADING_2"
  >
    A
  </Styled(Styled(Text))>
</Styled(View)>
`;
